OrbitGl/App.cpp

```diff
@@ -677,9 +685,20 @@ outcome::result<void, std::string> OrbitApp::OnLoadSession(
 
 //-----------------------------------------------------------------------------
 void OrbitApp::LoadSession(const std::shared_ptr<Session>& session) {
-  if (SelectProcess(Path::GetFileName(session->m_ProcessFullPath))) {
-    Capture::GSessionPresets = session;
+  if (Capture::GTargetProcess->GetFullPath() == session->m_ProcessFullPath) {
+    // In case we already have the correct process selected
+    GOrbitApp->LoadModulesFromSession(Capture::GTargetProcess, session);
+    return;
+  }
+  bool select_result =
+      SelectProcess(Path::GetFileName(session->m_ProcessFullPath));
+  if (!select_result) {
```

Comment:
I think you can just do !SelectProcess here... the result is not used anywhere else is it?

Reply:
yes, this was just for readability purposes.. but i guess it is also clear in the short form. Will change